modules/dynamic-plug-ins-get-started.adoc

```diff
@@ -14,9 +14,11 @@ To get started using the dynamic plug-in, you must set up your environment to wr
 
 .Procedure
 
-. Visit this link:https://github.com/spadgett/console-plugin-template[repository] containing a template for creating plug-ins.
+. Clone the template repository for creating plug-ins by running the following command: 
```

Comment:
I'm confused by this change. How is the repository created if there is no template?

I think there is some confusion between the GitHub template and OpenShift template. These are two different things. We've only removed the OpenShift template, not the GitHub template.